library/src/scala/quoted/Quotes.scala

```diff
@@ -2775,6 +2775,9 @@ trait Quotes { self: runtime.QuoteUnpickler & runtime.QuoteMatching =>
       extension (self: TypeRef)
         def isOpaqueAlias: Boolean
         def translucentSuperType: TypeRepr
+        /** The type bounds of the referenced type. */
```

Comment:
If I have a TypeRef to a class type, then its underlying will be a ClassInfo which I believe we don't want to expose in the reflection API?

Reply:
We do not expose those.